packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css

```diff
@@ -0,0 +1,88 @@
+.awards-section-hack-aubg-container {
+    padding-bottom: 5%;
```

Comment:
Please change all paddings in the css file to margins instead. Good job on using % and em for responsiveness efficiency.

Reply:
✓